TestHelpers.Tests/MockDirectoryInfoTests.cs

```diff
@@ -283,5 +283,346 @@ public void MockDirectoryInfo_ToString_ShouldReturnDirectoryName()
             // Assert
             Assert.AreEqual(directoryPath, str);
         }
+
+        [Test]
+        public void MockDirectoryInfo_Attributes_ShouldReturnDefaultAttributeIfNotExists()
+        {
+            ExecuteDefaultValueTest((d) => d.Attributes, MockFileData.NullObject.Attributes);
```

Comment:
This might start a bit of an opinion war, but I'm not a huge fan of this when writing tests. I dont so much mind hiding setup stuff behind a 

```csharp
var directoryInfo = CreateDefaultDirectoryInfo();
directoryInfo.Attributes // whatever act stuff
Assert(//whatever assert)
```

But this approach seems a little confusing to read. Not sure what everyone else's thoughts are.

Reply:
I tend to agree that in *test* code favoring readability at the expense of redundancy (code duplication) is not a bad thing. I'll go up ahead and update this.